src/RulesTest/Library/LocalizedControlTypeIsNotCustom.cs

```diff
@@ -10,51 +11,101 @@ public class LocalizedControlTypeIsNotCustom
     {
         private Axe.Windows.Rules.IRule Rule = new Axe.Windows.Rules.Library.LocalizedControlTypeIsNotCustom();
 
-        [TestMethod]
-        public void TestControlTypeIdIsNotCustomControl()
+        private MockA11yElement CreateElementExpectedToMatchCondition()
```

Comment:
Maybe use a test fixture and give this a `BeforeEach` tag?

Reply:
Perhaps if there were multiple setup steps. But as is, I don't think it adds much.